app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,23 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions()
       GOVUK.analytics.trackPageview();
+    },
+
+    'setCustomDimensions': function() {
+      var eligibility;
+      
+      if (/^\/suppliers\/opportunities\/(\d+)\/responses\/create$/.test(GOVUK.GDM.analytics.location.pathname())) {
```

Comment:
It would be better to have one single reference for the Python and JavaScript to use (perhaps a config file) but at the moment it's all kept inside the app so we can't do this.

@risicle is right and I've done projects before where the JavaScript and server-side language both point at a single data source (JSON usually) for shared information.

Might be worth looking at in future as we can output all the paths registered to the app so it could be a build task which produces a JSON file the JavaScript could reference.